### DISCUSSION TITLE
how to decrease RMS training error of force in lcurve.out?

### DISCUSSION THREAD

**Discussion by Wangmumu0403** [deepmodeling/deepmd-kit#1341] [General]
I have tried to increase the NN neuron to [240 240 240] and incrase the start_pref_f to 2000 and limit_pref_f is 1.2  and axis_neoron to 1.6 ; while these operations not work for rmse_f_trn in lcurve.out. 
At the same time . I also check my Dataset, I want to ask if the force's fluctuation range  will influence the  rmse_f_trn?
which part will help for my decrease the rmse_f_trn ?

#########
Here is my  rmse_f_trn in lcurve.out. 
![Screenshot from 2021-12-08 10-01-34](https://user-images.githubusercontent.com/95537862/145136171-8689ce10-1787-405b-be56-dd614812a693.png)

and .json file
#[input.json.docx](https://github.com/deepmodeling/deepmd-kit/files/7672815/input.json.docx)
and dataset Raw file's force file 's the first column
![Screenshot from 2021-12-07 17-54-33](https://user-images.githubusercontent.com/95537862/145136426-0c7747bb-53f4-400d-a7d1-2f681631ad95.png)

**Comment by Wangmumu0403** [comment]
Q3: Does the discrete point will influence the force training?

**Comment by njzjz** [comment]
I notice your `rcut_smth` is larger than `rcut`, which may result in unusual behavior.  `rcut_smth` should be less than `rcut`.

**Reply by Wangmumu0403** [reply]
Thanks ! I will test this parameter again .
